automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -97,13 +98,38 @@ class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Wo
           }
         }
       }
+
+      "a cloned workspace" - {
+        "should default to visible to 'all users' in Library Attributes" in withWebDriver { implicit driver =>
+          //create/publish a workspace
+          val curatorUser = UserPool.chooseCurator
+          implicit val curatorAuthToken: AuthToken = curatorUser.makeAuthToken()
+          withWorkspace(namespace, "PublishSpec_curator_publish_") { wsName =>
+            withCleanUp {
+              val data = LibraryData.metadata + ("library:datasetName" -> wsName)
+              api.library.setLibraryAttributes(namespace, wsName, data)
+              register cleanUp api.library.unpublishWorkspace(namespace, wsName)
+              api.library.publishWorkspace(namespace, wsName)
+              //clone workspace
+              withSignIn(curatorUser) { listPage =>
+                val workspaceNameCloned = "PublishSpec_curator_publish_cloned" + randomUuid
+                val workspaceSummaryPage = new WorkspaceSummaryPage(namespace, wsName).open
+                register cleanUp api.workspaces.delete(namespace, workspaceNameCloned)
+                workspaceSummaryPage.cloneWorkspace(namespace, workspaceNameCloned)
+                //Verify default group "All users"
+                val accessGroup = Orchestration.library.getDiscoverableGroups(namespace, workspaceNameCloned)
+                accessGroup.size shouldBe 0
```

Comment:
The endpoint getDiscoverableGroups checks what groups can see the dataset in the library, for open access the endpoint returns [] group that translates to All Users. It can be done through the UI but it is faster through API

Reply:
Please add this explanation as a code comment here.